### ISSUE TITLE
Android: NotificationReceivedHandler & NotificationOpenedHandler firing

### ISSUE THREAD

**Issue by xtt13** [cleverpush/cleverpush-flutter-sdk#5]
Hi @gbhrdt ,

I got the exact same Problem like in the issue #3 but just on Android. iOS works fine! 
In-App handler not firing in foreground or background.

Here is the log from my debug console. Everything seems ok.
```
D/CleverPush(23360): onMessageReceived
D/CleverPush(23360): Notification data: {subscription={"_id":"#######","syncedAt":"2020-07-22T12:55:54.448Z"}, notification={"mediaUrl":null,"customData":{"tour-id":"6730"},"title":"Test","expiresAt":null,"url":"https:\/\/google.com","createdAt":"2020-07-22T13:52:51.850Z","chatNotification":null,"_id":"gREJLrPRDg8dBdgt5","iconUrl":"https:\/\/static.cleverpush.com\/notification\/icon\/WmRdRX829zSAuhDp7.png","tag":"gREJLrPRDg8dBdgt5","text":null,"category":null,"actions":null}}
```

Flutter Doctor:
```
[✓] Flutter (Channel stable, v1.17.5, on Mac OS X 10.15.2 19C57, locale de-DE)
    • Flutter version 1.17.5 at /Users/spina/flutter
    • Framework revision 8af6b2f038 (3 weeks ago), 2020-06-30 12:53:55 -0700
    • Engine revision ee76268252
    • Dart version 2.8.4

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/spina/Library/Android/sdk
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3, Build version 11C29
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 42.1.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.47.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.12.2

[✓] Connected device (1 available)
    • IN2013 • e91431f9 • android-arm64 • Android 10 (API 29)

• No issues found!
```

**Comment by gbhrdt** [comment]
Hi @xtt13  Thanks for the bug report! Version 1.2.0 has been released which should solve this issue.

**Comment by gbhrdt** [comment]
Be sure to change minSdkVersion to atleast 17 (was 16 before in our example project)

**Comment by xtt13** [comment]
In-App handler are firing now but notifications are not shown anymore on my device. I tested it on a real device and on my emulator. 

I changed minSdkVersion to 18 because of another plugin.

**Comment by gbhrdt** [comment]
You mean notifications are shown when your app is in the background but not when in the foreground? This is intended behaviour, but we can add a setting for this.

**Comment by xtt13** [comment]
Ahh i get it! In the background they work as desired. In the foreground would be cool because "welcome-notifications" are not shown after I opted-in. This was working in the previous release.

**Comment by gbhrdt** [comment]
We released 1.2.1 which now has the same behavior like previous versions.

**Comment by xtt13** [comment]
Thank you!
